### PR TITLE
General Python Fixment

### PR DESCRIPTION
**Pull request by ghost**
- Files:
   * test/networkd-test.py 
   * test/sysv-generator-test.py
- Followed PEP Rules:
   * PEP8
   * PEP3101
- Changes: 
    * Imports re-ordered by Alphabetical Standarts for following PEP8
    * Old type string formattings (" example %s " % exampleVar ) re-writed as new type string formattings ( " example {} ".format(exampleVar) ) for following PEP3101
---
[PEP8](https://www.python.org/dev/peps/pep-0008/)

[PEP3101](https://www.python.org/dev/peps/pep-3101/)

### CODE REVIEW

**Comment by BTaskaya** [comment]
My changes doesn't related with semaphoreCI failing. It is related with installing some packages
image

**Comment by BTaskaya** [comment]
@keszybz  can you re-view it?

**Comment by yuwata** [comment]
The CI semaphoreci was broken when you submitted this. But all the other ubuntu CIs also failed, although they passed for the other PRs at that time. Could you check the failures and your commits?

**Comment by yuwata** [comment]
OK. So, please squash the second and third commits. We usually request patches in complete form and do not require the history of PRs. When you find errors or are requested to change the commits, then please force-push new version. Thank you.

**Comment by BTaskaya** [comment]
@yuwata squashed

**Comment by yuwata** [comment]
Then, please improve the commit title and message. 'General Python Fixment' is too abstract. I do not have enough knowledge about the PEP rules, but I guess mentioning them in the commit message seems fine, as you already explained in this PR.

I want to confirm that in the commit title you said "fixment", but it seems that this does not fix any bugs but just improves(?) coding style. Is it true?

**Comment by BTaskaya** [comment]
@yuwata ok fixed.

**Comment by yuwata** [comment]
Hmm... Why do not you just copy and paste (and fix typos in) the text which you wrote for explaining this PR to the commit message?

You have changed some coding style of the scripts. But we do not have any explicit coding style guide for Python codes. So, I think it is important to mention that the changes are based on the PEP rules.

**Comment by BTaskaya** [comment]
@yuwata 🆗

File: test/networkd-test.py [hunk 1]
```diff
@@ -130,7 +130,7 @@ def assert_link_states(self, **kwargs):
 
         # Wait for the requested interfaces, but don't fail for them.
         subprocess.call([NETWORKD_WAIT_ONLINE, '--timeout=5'] +
-                        ['--interface=%s' % iface for iface in kwargs])
+                        ['--interface={}'.format(iface for iface in kwargs)])
```
Comment (keszybz) [inline test/networkd-test.py]: Hey, that's not right! This changes a list comprehension to a list with a single item like (`['--interface=<generator at 0x...>']`).

**Comment by keszybz** [comment]
I think if you just say that the imports are sorted and "new" string formatting is used. Nothing more is needed.

File: test/networkd-test.py (hunk 1 above)
Comment (ghost) [reply test/networkd-test.py]: Sorry i missed it. I just simply change position of ")"

**Comment by BTaskaya** [comment]
@keszybz @yuwata 🆗

File: test/networkd-test.py [hunk 2]
```diff
@@ -653,7 +652,7 @@ def test_transient_hostname(self):
             self.addCleanup(subprocess.call, ['umount', '/etc/hostname'])
         subprocess.check_call(['systemctl', 'stop', 'systemd-hostnamed.service'])
 
-        self.create_iface(dnsmasq_opts=['--dhcp-host=%s,192.168.5.210,testgreen' % self.iface_mac])
+        self.create_iface(dnsmasq_opts=['--dhcp-host={},192.168.5.210,testgreen'.format(self.iface_mac]))
```
Comment (keszybz) [inline test/networkd-test.py:655]: Pfff, please don't send PR that haven't been tested. This is a syntax error.

File: test/networkd-test.py [hunk 3]
```diff
@@ -688,7 +687,7 @@ def test_transient_hostname_with_static(self):
             self.writeConfig('/etc/hostname', orig_hostname)
         subprocess.check_call(['systemctl', 'stop', 'systemd-hostnamed.service'])
 
-        self.create_iface(dnsmasq_opts=['--dhcp-host=%s,192.168.5.210,testgreen' % self.iface_mac])
+        self.create_iface(dnsmasq_opts=['--dhcp-host={},192.168.5.210,testgreen'.format(self.iface_mac]))
```
Comment (keszybz) [inline test/networkd-test.py:690]: Here too.

**Pull request closed without merging**